compat/media/camera_service.cpp

```diff
@@ -45,7 +45,7 @@ int main(int argc, char** argv)
     // for creating a new IMediaRecorder (MediaRecorder) instance over Binder
     MediaRecorderFactory::instantiate();
     // Enable audio recording for camera recording
-    CameraRecordService::instantiate();
+    // CameraRecordService::instantiate();
```

Comment:
These changes aren't pretty

Reply:
good point, why are these changes necessary?

Reply:
I'll ask the author of the patch set but a quick test build without the line being commented resulted in failure due to `use of undeclared identifier 'CameraRecordService';`.

Reply:
The header was removed in commit https://github.com/libhybris/libhybris/commit/ffcf03a248ad739f83ae55966da9c0599b0613d8 but I couldn't find the header or other files anywhere in Android source tree and web searches didn't help either, those files were only in libhybris.

Reply:
How should we proceed here? Any specific info you need?

Reply:
this probably needs to be inside some ifdefs?

Reply:
halium-5.1 has `camera_record_service.cpp` in https://github.com/Halium/android_frameworks_av/tree/halium-5.1/media/libmedia but halium-7.1 doesn't have that. It rather sounds like these changes have to applied to the halium-7.1 as well instead of patching the libhybris side

Reply:
camera_record_service.h Seems to be Ubuntu specific? All (5) references point to that? Seeing it's compat, isn't this Android 5.1 only anyway? https://github.com/search?q=CAMERA_RECORD_SERVICE_H&type=Code

Reply:
Yes, based on copyright header in the files those were added by Canonical.